multiqc/modules/truvari/bench.py

```diff
@@ -0,0 +1,234 @@
+            "title": "FN",
+            "description": "Number of the base VCF calls non-matching the comp VCF",
+            "scale": "BuGn",
+            "format": "{:,.d}",
+            "min": 0,
+        }
+
+        # Reuse info from bench_headers but hide by default
+        keys["precision"] = dict(bench_headers["precision"], hidden=True)
+        keys["recall"] = dict(bench_headers["recall"], hidden=True)
+        keys["f1"] = dict(bench_headers["f1"], hidden=True)
+
+        keys["base cnt"] = {
+            "title": "Base",
+            "description": "Number of calls in the base VCF",
+            "scale": "Oranges",
+            "format": "{:,.d}",
+            "min": 0,
+        }
+        keys["call cnt"] = {
+            "title": "Call",
+            "description": "Number of calls in the comp VCF",
+            "scale": "Blues",
+            "format": "{:,.d}",
+            "min": 0,
+        }
+        keys["TP-call_TP-gt"] = {
+            "title": "TP-call with GT match",
+            "description": "TP-call with genotype match",
+            "hidden": True,
+            "scale": "Greens",
+            "format": "{:,.d}",
+            "min": 0,
+        }
+        keys["TP-call_FP-gt"] = {
+            "title": "TP-call w/o GT match",
+            "description": "TP-call without genotype match",
+            "hidden": True,
+            "scale": "YlGn",
+            "format": "{:,.d}",
+            "min": 0,
+        }
+        keys["TP-base_TP-gt"] = {
+            "title": "TP-base with GT match",
+            "description": "TP-base with genotype match",
+            "hidden": True,
+            "scale": "RdPu",
+            "format": "{:,.d}",
+            "min": 0,
+        }
+        keys["TP-base_FP-gt"] = {
+            "title": "TP-base w/o GT match",
+            "description": "TP-base without genotype match",
+            "hidden": True,
+            "scale": "Purples",
+            "format": "{:,.d}",
+            "min": 0,
+        }
+        keys["gt_concordance"] = {
+            "title": "GT concordance",
+            "description": "Genotype concordance. Definition: TP-call with GT / (TP-call with GT + TP-call w/o GT)",
+            "hidden": True,
+            "scale": "GnBu",
+            "suffix": "%",
+            "modify": lambda x: x * 100,
+        }
+
+        self.add_section(
+            name="Truvari bench",
+            anchor="truvari-bench",
+            description="Concordance statistics parsed from the output from <code>truvari bench</code>.",
+            plot=table.plot(data, keys, {"id": "truvari-bench-summary"}),
+        )
+        # Generate shuffled list of colors to label samples
+        colors = list(mcolors.CSS4_COLORS)
+        random.shuffle(colors)
+
+        # Check that not too many samples for scatter plot
+        if len(data) > len(colors):
+            log.debug("Too many samples ({}) to generate scatter plot.".format(len(data)))
+            return len(data)
```

Comment:
I think it's not a good idea to limit the number of samples here. If we scale up, it's okay if some samples have the same color, but we still should support the arbitrary number of samples. Probably in the future MultiQC will be smarter with the large numbers of samples and generate more efficient plots e.g. adding interactivity for only outliers.

Reply:
Yes, not sure why I though this was a good idea. It is removed now!